http/vibe/http/fileserver.d

```diff
@@ -311,7 +312,43 @@ private void sendFileImpl(scope HTTPServerRequest req, scope HTTPServerResponse
 	if ("Content-Encoding" in res.headers && isCompressedFormat(mimetype))
 		res.headers.remove("Content-Encoding");
 	res.headers["Content-Type"] = mimetype;
-	res.headers["Content-Length"] = to!string(dirent.size);
+	res.headers.addField("Accept-Ranges", "bytes");
+	ulong rangeStart = 0;
+	ulong rangeEnd = 0;
+	auto prange = "Range" in req.headers;
+
+	if (prange) {
+		auto range = (*prange).chompPrefix("bytes=");
+		if (range.canFind(','))
+			throw new HTTPStatusException(HTTPStatus.notImplemented);
+		auto s = range.split("-");
+		// https://tools.ietf.org/html/rfc7233
+		// Range can be in form "-\d", "\d-" or "\d-\d"
+		if (s[0].length) {
+			rangeStart = s[0].to!ulong;
+			rangeEnd = s[1].length ? s[1].to!ulong : dirent.size;
+		} else if (s[1].length) {
```

Comment:
Uh well this PR is merged already and I recreated the repo so I can't edit this anymore, I will make a separate PR.